skimage/restoration/_denoise.py

```diff
@@ -56,8 +56,15 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
 
     """
     mode = _mode_deprecations(mode)
-    return _denoise_bilateral(image, win_size, sigma_range, sigma_spatial,
-                              bins, mode, cval)
+
+    offset = np.min(image)
+    offset_img = image - offset
+    if not offset_img.any():
+        # if all value identical, returns the same image
+        return image
+    else:
+        return _denoise_bilateral(offset_img, win_size, sigma_range, sigma_spatial,
```

Comment:
This is changing the existing behavior and I don't see a reason why to subtract the min from the image. If you want to test for uniqueness, do a `image.min() - image.max() == 0` test.